### PR TITLE
Add maintainers list

### PR DESCRIPTION
**Pull request by belochub**

### CODE REVIEW

File: doc/maintainers.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| `metarhia/impress-cli`                 | `impress-cli`                 | [belochub][]     |
+| `metarhia/jstp`                        | `@metarhia/jstp`              | [belochub][]     |
+| `metarhia/mdsf`                        | `mdsf`                        | [belochub][]     |
+| `metarhia/metacom`                     | `metacom`                     | [tshemsedinov][] |
+| `metarhia/metadoc`                     | `@metarhia/doc`               | [lundibundi][]   |
+| `metarhia/metalog`                     | `metalog`                     | [nechaido][]     |
+| `metarhia/metaschema`                  | `metaschema`                  | [nechaido][]     |
+| `metarhia/metastreams`                 | `metastreams`                 | [lundibundi][]   |
+| `metarhia/metasync`                    | `metasync`                    | [nechaido][]     |
+| `metarhia/metatests`                   | `metatests`                   | [lundibundi  ][] |
```
Comment (tshemsedinov) [inline doc/maintainers.md]:
```suggestion
| `metarhia/metatests`                   | `metatests`                   | [lundibundi][]   |
```

File: doc/maintainers.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| `metarhia/metadoc`                     | `@metarhia/doc`               | [lundibundi][]   |
+| `metarhia/metalog`                     | `metalog`                     | [nechaido][]     |
+| `metarhia/metaschema`                  | `metaschema`                  | [nechaido][]     |
+| `metarhia/metastreams`                 | `metastreams`                 | [lundibundi][]   |
+| `metarhia/metasync`                    | `metasync`                    | [nechaido][]     |
+| `metarhia/metatests`                   | `metatests`                   | [lundibundi][] |
+| `metarhia/protocol`                    | `mhp`                         | [nechaido][]     |
+| `metarhia/remark-preset-lint-metarhia` | `remark-preset-lint-metarhia` | [belochub][]     |
+| `metarhia/sandboxed-fs`                | `sandboxed-fs`                | [belochub][]     |
+| `metarhia/tickplate`                   | `tickplate`                   | [tshemsedinov][] |
```
Comment (lundibundi) [inline doc/maintainers.md]: Perhaps also add `jstp-java`? (in case, the relevant package is https://bintray.com/metarhia/maven/jstp)

**Comment by tshemsedinov** [comment]
It would be great to add links to all github and npm repos @belochub WDYT?

File: doc/maintainers.md (hunk 2 above)
Comment (belochub) [reply doc/maintainers.md]: It is not an NPM package, this table is only for packages that are published to NPM.
I can add a separate table below for Java packages, WDYT?

Comment (tshemsedinov) [reply doc/maintainers.md]: Or second table for all outside of NPM

**Comment by belochub** [comment]
@tshemsedinov I've added the links.

**Pull request merged**